internal/core/drand_beacon_control.go

```diff
@@ -541,25 +544,21 @@ func (bp *BeaconProcess) chainInfoFromPeers(ctx context.Context, privGateway *ne
 // sendProgressCallback returns a function that sends SyncProgress on the
 // passed stream. It also returns a channel that closes when the callback is
 // called with a beacon whose round matches the passed upTo value.
-func (bp *BeaconProcess) sendProgressCallback(
-	ctx context.Context,
-	stream drand.Control_StartFollowChainServer, // not ideal since we also reuse it for the StartCheckChain
-	upTo uint64,
-	info *chain2.Info,
+func (bp *BeaconProcess) sendProgressCallback(ctx context.Context,
```

Comment:
this formatting looks a bit funky - newline per param would be nicer

Reply:
Granted